website/docs/concepts/html/introduction.mdx

```diff
@@ -17,6 +17,7 @@ The `html!` macro allows you to write HTML and SVG code declaratively. It is sim
    [fragments](./fragments.mdx) or [iterators](./../html/lists.mdx))
 2. An empty `html! {}` invocation is valid and will not render anything
 3. Literals must always be quoted and wrapped in braces: `html! { <p>{ "Hello, World" }</p> }`
+4. The `html!` macro will make all tag names lower case. To use upper case characters (which are required for some SVG elements) use [dynamic tag names](./html/elements#dynamic-tag-names): `html! { <@{"myTag"}></@> }`
```

Comment:
```suggestion
4. The `html!` macro will make all tag names lower case. To use upper case characters (which are required for some SVG elements) use [dynamic tag names](concepts/html/elements.mdx#dynamic-tag-names): `html! { <@{"myTag"}></@> }`
```

Could this refer to the `.mdx` file instead, it's more direct when searching internal vs external links. Using a relative link (rel to the docs root) should also make translations work correctly.